src/main/java/ti4/service/info/ListPlayerInfoService.java

```diff
@@ -241,6 +242,9 @@ public static void displayerScoringProgression(Game game, boolean onlyThisGameOb
             }
             msg.append(representSecrets(game)).append("\n");
             msg.append(representSupports(game)).append("\n");
+            if (gameHasTransferablePoints(game)) {
+                msg.append(representTransferablePoints(game)).append("\n");
+            }
```

Comment:
These functions print the "Transferable" points in the game (if any). These are points which can be lost or transferred by some means. `Political Censure` barely makes the cut; any others I missed, particularly in homebrew content?